CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing to Arbor
+
+First off, thank you for considering contributing to Arbor! It's people like you that
+make Arbor such a great tool. Feel welcome and read the following sections in order to
+know how to ask questions and how to work on something.
+
+## Types of contributions
+
+There are many ways to contribute, from writing tutorials or blog posts, improving the
+documentation, submitting bug reports and feature requests or writing code which can be
+incorporated into Arbor itself.
+
+If you come across a bug in Arbor, please [file an issue on the GitHub issue tracker](https://github.com/arbor-sim/arbor/issues/new).
```

Comment:
Do we have a template? The usual procedure seems to be to require a small explanation of what the user did, what what happend, and what was the desired outcome (instead).

Reply:
We have [issue templates](https://github.com/arbor-sim/arbor/tree/master/.github/ISSUE_TEMPLATE), based on the Github provided templates.